config/databases/mongo.rb

```diff
@@ -0,0 +1,3 @@
+require 'slack-ruby-bot-server/models/team/mongo.rb'
```

Comment:
Just for consistency sake it's easier if the ENV variable keys the names, if `DATABASE_ADAPTER=mongoid` then this file should be called `database_adapters/mongoid.rb`.